apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V20241101111900986__create_samhsa_2_tables.sql

```diff
@@ -0,0 +1,119 @@
+DROP TYPE IF EXISTS tag_code CASCADE;
+CREATE TYPE tag_code AS ENUM('R', '42CFRPart2');
+
+CREATE TABLE IF NOT EXISTS ccw.hha_tags (
+    tag_id BIGINT,
+    clm_id BIGINT NOT NULL,
+    code TAG_CODE,
```

Comment:
Do we need a `NOT NULL` constraint on the `code` column as well?

Reply:
Yes, probably should.